physlr/physlr.py

```diff
@@ -1467,19 +1483,24 @@ def physlr_split_minimizers(self):
             sys.exit(msg)
         g = self.read_graph([self.args.FILES[0]])
         bxtomxs = self.read_minimizers([self.args.FILES[1]])
+        molto0 = {mol : 0 for mol in g.nodes()}
+        bxtocount = self.count_molecules_per_bx(molto0)
```

Comment:
I totally understand just doing this transformation to try to use the existing function, but I think it is really cleaner to just make a new function that does exactly what you want to do instead. (ie. iterate over the nodes in the graph, and count the number of molecules)

Who knows, having that function might actually be helpful for us in the future, and it isn't too many lines of code.

Reply:
done